pkgs/by-name/te/tenere/package.nix

```
@@ -0,0 +1,26 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "tenere";
+  version = "0.11.2-unstable-2024-12-05";
+  src = fetchFromGitHub {
+    owner = "pythops";
+    repo = "tenere";
+    rev = "0f3181ab23066aa69aa4fec387a7e16578078179";
+    hash = "sha256-HKPCX0bmXkB3LwvgE1li3dlWTgpW5CXuWZNq3mFY6FY=";
+  };
+  cargoHash = "sha256-u6QCs3RaLeRoVfhRzUG7wwx7kd0O3KyiC5vhZ3mE7UM=";
+
+  requiredSystemFeatures = [ "big-parallel" ]; # for fat LTO from upstream
+
+  meta = {
+    description = "Terminal interface for large language models (LLMs)";
+    homepage = "https://github.com/pythops/tenere";
+    license = lib.licenses.gpl3;
```

```suggestion
    license = lib.licenses.gpl3Only;
```

https://github.com/pythops/tenere/blob/0f3181ab23066aa69aa4fec387a7e16578078179/Cargo.toml#L5
https://github.com/pythops/tenere/blob/0f3181ab23066aa69aa4fec387a7e16578078179/README.md?plain=1#L299

---

Thank you, good catch